state-chain/pallets/cf-elections/src/electoral_systems/block_witnesser/block_processor.rs

```diff
@@ -355,13 +349,7 @@ pub(crate) mod test {
 					},
 				}
 			}
-			chosen.into_values().collect()
```

Comment:
I think this hook implementation can now be empty since we don't want to return anything from this hook. Instead we could unit-test the the `deduplicate_events()` function (if we don't do that already...).